src/context/GameContext.tsx

```diff
@@ -85,26 +100,36 @@ export function GameContextProvider({ children, deckId }) {
   }, [deckId, drawOnGameStart]);
 
   useEffect(() => {
-    if (roundEnded.croupier) {      
+    if (roundEnded.croupier) {
+      let newMessage: string;
+      let currentRoundWinner: winner;
```

Comment:
```suggestion
      const currentRoundWinner = gameContextHelpers.findWhoWonRound(points.playerPoints, points.croupierPoints;
```

i usunąć ustawianie w pojedynczysz switch